bin/oneoff/applab_datasets/upload_csv_to_firebase

```diff
@@ -26,7 +26,7 @@ def parse_csv(filename)
     records[id] = record.to_h.to_json
     id += 1
   end
-  return records, table.headers
+  return records, table.headers.unshift('id')
```

Comment:
(not blocking - just for context) What does `.unshift('id')` do here?

Reply:
Prepends `'id'` to the front of the list. We want to make sure `'id'` is the first column shown in the table, and this was the easiest way I could think to ensure that.